### ISSUE TITLE
Link to preprocessed data from tutorial is incomplete

### ISSUE THREAD

**Issue by papajohn** [google/seq2seq#45]
In [1], linked from [2], the test sets are not subword encoded. In addition, the merges file from BPE is not included in the archive. Therefore, a model trained on the subword-encoded training set cannot be evaluated without downloading the raw data and training the BPE model from scratch.

[1] https://drive.google.com/open?id=0B_bZck-ksdkpREE3LXJlMXVLUWM
[2] https://google.github.io/seq2seq/nmt/

```
$ tar -tzvf wmt16_en_de.tar.gz 
-rw-r--r-- dennybritz/eng 279423 2017-03-07 04:31 vocab.bpe.32000
-rw-r--r-- dennybritz/eng 778882202 2017-03-07 04:29 train.tok.clean.bpe.32000.de
-rw-r--r-- dennybritz/eng 673102722 2017-03-07 04:21 train.tok.clean.bpe.32000.en
-rw-r--r-- dennybritz/eng    393971 2017-03-07 02:43 newstest2016.tok.de
-rw-r--r-- dennybritz/eng    354403 2017-03-07 02:44 newstest2016.tok.en
-rw-r--r-- dennybritz/eng    279171 2017-03-07 02:43 newstest2015.tok.de
-rw-r--r-- dennybritz/eng    253703 2017-03-07 02:44 newstest2015.tok.en
-rw-r--r-- dennybritz/eng    410292 2017-03-07 02:43 newstest2014.tok.de
-rw-r--r-- dennybritz/eng    377491 2017-03-07 02:44 newstest2014.tok.en
-rw-r--r-- dennybritz/eng    399549 2017-03-07 02:43 newstest2013.tok.de
-rw-r--r-- dennybritz/eng    349480 2017-03-07 02:44 newstest2013.tok.en
```

What's missing are the `bpe.32000` merges file and `newstest*.tok.bpe.32000.*` subword-encoded test set files. (Not hard to regenerate, but might as well include them for future folk.)

**Comment by dennybritz** [comment]
Yup, that data should be there. Will uploaded an updated version.

**Comment by dennybritz** [comment]
Updated data path: https://drive.google.com/open?id=0B_bZck-ksdkpM25jRUN2X2UxMm8

**Comment by timolohrenz** [comment]
Hey, seems like the preprocessed data is not there anymore. Would it be possible to share an updated link?

**Comment by SCUTQiaoRui** [comment]
您的邮件已收到，，，我会尽快回复的，，，

**Comment by devesh28** [comment]
Hi, The data is still available, can developers post the data link.

**Comment by SCUTQiaoRui** [comment]
您的邮件已收到，，，我会尽快回复的，，，
